### DISCUSSION TITLE
v3.0.0b5

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#585] [Announcements]
## What's Changed
* Improve auto update performance by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/584
* NOTE: v3.0.b3/4 is a dead end, it doesn't auto update due to a bug in esp32 framework.  Gonna have to plug them into a pc and flash via usb :(

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v3.0.0b4...v3.0.0b5

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.0.0b5'>v3.0.0b5</a>.</em>

**Comment by onlyoneme** [comment]
Mine has downgraded from b3 to 2.2.9 after disabling pre-released versions.

**Reply by DTTerastar** [reply]
Wow!  None of my m5atom's would validate the update.  Glad to hear there is hope for not fixing them manually :)
